gen_install.sh

```diff
@@ -31,12 +48,9 @@ echo "INSTALLING DEPENDENCIES NOT FOUND IN ROSDEP"
 #pip install pyqtgraph --user
 #I think I have replaced this by adding a symlink:
 #python flo_face/teensy/src/serial_coms/computer/python/serial-coms/setup.py install --user
-# Mutagen has dropped python 2 support. Last supported version was 1.43.0:
-sudo apt-get -qq install -y python-pip
-pip install 'mutagen==1.43.0' --user -q
+# Mutagen has dropped python 2 support. Last supported version was 1.43.0: mutagen moved up into if statement to account for ubuntu 20, also python3 supports latest version of mutagen.
```

Comment:
can just move this comment up to line 28

Reply:
Okay